packages/wrangler/src/__tests__/deploy.test.ts

```diff
@@ -6104,11 +6077,13 @@ addEventListener('fetch', event => {});`
 
 		describe("[logfwdr]", () => {
 			it("should support logfwdr bindings", async () => {
-				fs.writeFileSync("./message.capnp.compiled", "compiled capnp messages");
+				(callCapnp as jest.MockedFunction<typeof callCapnp>).mockImplementation(
+					() => Buffer.from("compiled capnp messages")
+				);
 
 				writeWranglerToml({
 					logfwdr: {
-						schema: "./message.capnp.compiled",
+						schema: "./message.capnp",
```

Comment:
Lets update this field name from `schema` to `capnp_schema` to match unsafe bindings.